src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -421,7 +421,7 @@ public function render()
 
                 if ($isFirstRow) {
                     $this->renderRowSeparator(
-                        $isHeader ? self::SEPARATOR_TOP : self::SEPARATOR_TOP_BOTTOM,
+                        $isHeader || $horizontal ? self::SEPARATOR_TOP : self::SEPARATOR_TOP_BOTTOM,
```

Comment:
Isn't `$isHeader` here always `false` by the logic? Or do I miss a case?
```suggestion
                        $horizontal ? self::SEPARATOR_TOP : self::SEPARATOR_TOP_BOTTOM,
```

Reply:
lets check via CI

Reply:
Looks like it is ok, thanks @maxbeckers